### DISCUSSION TITLE
helmfile to wait for the subchart to be installed before the parent chart

### DISCUSSION THREAD

**Discussion by patsevanton** [helmfile/helmfile#497] [Questions] [answered]
Hello! Thanks for helmfile

Is it possible to use helmfile to wait for the subchart to be installed before the parent chart? Thanks!

**Comment by yxxhero** [comment]
@patsevanton subchart? maybe you can use needs feather or sub helmfile.

**Reply by patsevanton** [reply]
I try wait ready recources of subchart

For example: https://github.com/patsevanton/hook-weight-for-subchart

Cassanra deploy at the same time with main chart.

**Comment by yxxhero** [comment, marked as answer]
@patsevanton maybe should it be a feature of helm?

**Reply by patsevanton** [reply]
Ok. I created issue of helm. Thanks! Close discussions.
